### ISSUE TITLE
Date/Time format 

### ISSUE THREAD

**Issue by raduiman** [Akhliskun/firefox-infra-changelog#21]
## Short description:
Git and HG returns the date and time in different ways. 
eg.  Unix Timestamp from HG
To not be confusing we have to use a single date/time format. 

## Where to start:
We could start deciding what date/time format to use. For example the most popular date/time format is the following: YYYY-MM-DD Thh:mm:ss (eg 2018-11-02 T12:10:30) (follow the order of largest to smallest units)
-where:
     YYYY = four-digit year
     MM   = two-digit month (01=January, etc.)
     DD   = two-digit day of month (01 through 31)
     hh   = two digits of hour (00 through 23) (am/pm NOT allowed)
     mm   = two digits of minute (00 through 59)
     ss   = two digits of second (00 through 59)
-also we can add:
     TZD  = time zone designator (Z or +hh:mm or -hh:mm)
     Z = UTC (Coordinated Universal Time)

## Requirements:
* Don't use classes
* Runs on Py3.X
* Document the code

**Comment by bccrisan** [comment]
I think that would be the best to go with US date & time notation system.
TL;DR: MM-DD-YYYY Thh:mm:ss with TZD
or we could introduce the TZD directly in the Thh:mm:ss, thus showing the push earlier or later depending on the TZD.

**Comment by raduiman** [comment]
Since we have a general datetime format (YYY-MM-DD hh:mm:ss) on the entire project,  can we close this issue ?

**Comment by zsoltfay** [comment]
This has indeed been implemented and it is working. Safe to close.
